### ISSUE TITLE
ng-add expects path relative to /src/app

### ISSUE THREAD

**Issue by darvids0n** [ngx-formly/ngx-formly#2291]
**Description**
When running `ng add @ngx-formly/schematics`, the installation fails if the root app module is not in the default folder, even if you provide the --module option. You need to provide a relative path to `/src/app`, which may not even exist (eg in my case), for it to function.

**Minimal Reproduction**

```
ng new formly-schematic-test --minimal -g --routing=false --style=scss
cd formly-schematic-test; mv src/app src/root
# Edit src/main.ts to include AppModule from the proper path.
ng add @ngx-formly/schematics --module="root/app.module.ts"
```

**Expected result**

The schematic will find the specified module by searching from `project.sourceRoot`, or using the working directory plus the relative path provided, and successfully configure the project.

**Actual result**

The schematic fails with `Specified module does not exist`.
Installation succeeds if `--module="../root/app.module.ts"` is provided.

**Your Environment**

- Angular version: 9.1.9
- Formly version: 5.6.2

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.7](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.7.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by adelinor** [comment]
@aitboudad , 
The message `Specified module 'app' does not exist.` now pops up when the Angular app is created with the _default_ value for standalone option.
It might be good to rephrase this error message to indicate that the app needs to created with modules if this is a requirement.
